src/atwho.js

```diff
@@ -17,20 +17,21 @@ define([
 
     // stripped down version of http://codepen.io/ImagineProgramming/storydump/javascript-memoization-timeout
     var timed = function timed(f, timeout) {
-        var cache, time = 0;
+        var cache = {}, time = 0;
```

Comment:
Thanks I made this into a real plugin. I was originally just going for practicality since there should be a 1 to 1 from formuuid to vellum instance and it was getting thrown away every 500 ms anyways, but it's just as easy to do a deep reach into form to get the atwho data.

Looks like the caching was broken by a refactor. It should have been caching globally before but it wasn't caching at all